packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -90,8 +91,7 @@ class Plugin implements CLNPlugin {
       {required String event,
       required Future<Map<String, Object>> Function(Plugin, Map<String, Object>)
           onEvent}) {
-    notifications["event"] =
-        RPCCommand(name: "", usage: "", description: "", callback: onEvent);
+    notifications["event"] = RPCNotification(method: event, callback: onEvent);
```

Comment:
```suggestion
    notifications[event] = RPCNotification(method: event, callback: onEvent);
```

it is not a string here, but maybe was a old code